app/client/ui2018/cssVars.ts

```diff
@@ -111,7 +111,7 @@ export const vars = {
   labelTextBg:    new CustomProp('label-text-bg', '#FFFFFF'),
   labelActiveBg:  new CustomProp('label-active-bg', '#F0F0F0'),
 
-  controlMargin:  new CustomProp('normal-margin', '2px'),
+  controlMargin:  new CustomProp('normal-margin',  '2px'),
```

Comment:
Extra space not required.

```suggestion
  controlMargin:  new CustomProp('normal-margin', '2px'),
```

Reply:
OK then, you may resolve this conversation :)